data/init-mongo.js

```diff
@@ -1,4 +1,24 @@
 db = db.getSiblingDB("stamfordcenter");
+
+db.announcements.insertMany([
+	{
+		_id: "65212e74aac38c41507570b9",
+		title: "STIU Market",
+		content: "Get ready for upcoming STIU Market on Thursday!",
+		type: "notice",
+		createdAt: new Date("2024-01-20"),
+		expiresAt: new Date("2024-2-10"),
+	},
+	{
+		_id: "65212e74aac38c21507570f2",
+		title: "Final Exams",
+		content: "This is Week12 of the semester! Everyone ready for final exams?",
+		type: "warning",
+		createdAt: new Date("2024-01-20"),
+		expiresAt: new Date("2024-02-20"),
+	},
```

Comment:
can you add one more announcement that is already expired? And it should not be returned to the frontend